operations/research-and-development/product/product-technical-writing-team-handbook.md

```diff
@@ -104,13 +104,13 @@ For less urgent work, if no reply has been received within 48 hours, please ping
 
 We really want the community to own the docs as much as possible. In light of this, please mark your issue as “Help Wanted” so that community members are able to identify work that they’re able to assist with. 
 
-If your documentation request/issue applies to a repository other than mattermost/docs, you can use the Editor Review label and/or ping @justinegeffen.
+If your documentation request/issue applies to a repository other than mattermost/docs, you can use the \*\*Editor Review\*\*label and/or ping @justinegeffen or @amyblaise.
 
 ## General Contribution Workflow
 
 ### Jira Tickets
 
-The Technical Writing Jira board is located here: [https://mattermost.atlassian.net/secure/RapidBoard.jspa?rapidView=51](https://mattermost.atlassian.net/secure/RapidBoard.jspa?rapidView=51). Feature and release writing is planned three sprints in advance. This ensures that ad-hoc work can be taken on. 
+The Technical Writing Jira board is located here:c[https://mattermost.atlassian.net/secure/RapidBoard.jspa?rapidView=51](https://mattermost.atlassian.net/secure/RapidBoard.jspa?rapidView=51). Feature and release writing is planned three sprints in advance. This ensures that ad-hoc work can be taken on. 
```

Comment:
Did you intend to add the `c` here?

Reply:
Erroneous typo, thanks for catching it. I've fixed it!

Reply:
@amy